src/main/java/seedu/address/logic/commands/BatchDeleteCommand.java

```diff
@@ -19,32 +23,61 @@ public class BatchDeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "batchdelete";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": batch delete people whose policy expiry date "
-            + "is in the corresponding month and year. "
-            + "Parameters: "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": batch delete people whose policy: \n"
+            + "(i) expiry date is in the corresponding month and year OR \n"
+            + "(ii) belongs to a company.\n"
+            + "(i) Parameter: "
             + PREFIX_DELETE_MONTH + "MM-yyyy\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_DELETE_MONTH + "11-2022";
+            + PREFIX_DELETE_MONTH + "11-2022\n"
+            + "(ii) Parameter: "
+            + PREFIX_COMPANY + "COMPANY NAME\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_COMPANY + "Allianz";
```

Comment:
Nice to see you are extending the `batchdelete` feature to also batch delete by company!